### DISCUSSION TITLE
The 🇺🇸 recently announced it will open an embassy where?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#782] [Galaxy Trivia] [answered]
*answer with emoji*

**Comment by peterbaccaro** [comment, marked as answer]
The United States is on track to open a new embassy in **Tonga** this month, the top US diplomat for east Asia said, as part of the efforts to step up its diplomatic presence in the Pacific region to counter China.


The United States are also continuing to engage with Vanuatu and Kiribati about opening proposed new embassies in those countries.

**Comment by ParanoidUser** [comment]
I don't think I can find an emoji here.. 🙄

**Reply by peterbaccaro** [reply]
🇹🇴 TONGA
